project/src/main/java/com/google/sps/data/Popular.java

```diff
@@ -57,7 +58,7 @@ public void addToPopularList(String placeID) {
     // Load popular restaurants from Datastore
```

Comment:
maybe you want this to accept max_popular_restaurants so we don't return 1 million results to the user?